pxr/usdImaging/lib/usdImaging/delegate.h

```diff
@@ -267,6 +267,15 @@ class UsdImagingDelegate : public HdSceneDelegate, public TfWeakBase {
     USDIMAGING_API
     void SetWindowPolicy(CameraUtilConformWindowPolicy policy);
 
+    /// Sets display of unloaded prims as bounding boxes.
+    /// Unloaded prims will need bounds definition in order to see them.
```

Comment:
We should be more explicit here: Bounds for an unloaded prim will only be drawn if either the prim is a UsdGeomBoundable that has an authored _extent_ attribute, or if the prim is a \ref UsdPrim::IsModel() "model" and has an authored _extentsHint_ attribute (see UsdGeomModelAPI::GetExtentsHint()).